src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -1077,6 +1080,8 @@ private void UpdateCacheExpirationTime(KeyValueWatcher changeWatcher)
                 }
 
                 previousEndpoint = currentEndpoint;
+
+                _requestTracingOptions.IsFailoverRequest = true;
```

Comment:
Let's consider this concrete scenario, customer passes an endpoint (static) and we discover a replica endpoint (dynamic)
1. At a time point, the static one starts to fail, failover to the dynamic one, current code can handle the dynamic client as a failover
2. The dynamic one keeps failing and it refresh before its backoff time due, I think the dynamic client still need to have the failover tag in correlation context, but the current code will treat it as not a failover

Reply:
And we need to consider the loadbalance feature, the load balance requests should not be considered as failover

Reply:
If we don't consider loadbalancing, then we can have a very simple way to decide whether the request is from a failover: we can just check whether the current client is the first client from the list returned by `ConfigurationClientManager.GetClients`. But this way will break when load balancing is enabled.

Reply:
@RichardChen820 I sent an email to discuss this.